src/deploy/table_event_recommendation_policy.sql

```diff
@@ -0,0 +1,18 @@
+-- Deploy maevsi:table_event_recommendation_policy to pg
+
+BEGIN;
+
+GRANT SELECT, INSERT, DELETE ON TABLE maevsi.event_recommendation TO maevsi_account;
+
+ALTER TABLE maevsi.event_recommendation ENABLE ROW LEVEL SECURITY;
+
+-- Only allow selects by the current user.
+CREATE POLICY event_recommendation_select ON maevsi.event_recommendation FOR SELECT USING (
+  NULLIF(current_setting('jwt.claims.account_id', true), '')::UUID IS NOT NULL
+  AND
+  account_id = NULLIF(current_setting('jwt.claims.account_id', true), '')::UUID
+);
+
+-- TODO: policies for background process updating the recommendations
```

Comment:
I think we can remove this TODO here and just add the implementation once this task is actually being worked on. No need for a reminder for now.